runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -157,3 +158,30 @@ J9::ARM64::TreeEvaluator::newObjectEvaluator(TR::Node *node, TR::CodeGenerator *
    return targetRegister;
    }
 
+=======
```

Comment:
This line `=======` and the line `>>>>>>> AArch64: Implementation of ArrayLengthEvaluator` at the end of the function must be removed, too.